.github/actions/setup/action.yml

```diff
@@ -25,3 +25,5 @@ runs:
         yarn install --cwd example --frozen-lockfile
         yarn install --frozen-lockfile
       shell: bash
+      env:
+        SKIP_YARN_COREPACK_CHECK: '1'
```

Comment:
Skipping corepack for these tasks due to this https://github.com/gr4vy/gr4vy-react-native/actions/runs/8982545520/job/24670387490?pr=26#step:3:46. We don't need to use the same exact yarn version for linting, but will probably upgrade the supported one in the future.